src/azure-cli/azure/cli/command_modules/apim/_params.py

```diff
@@ -420,3 +420,64 @@ def load_arguments(self, _):
         c.argument(
             'service_name', options_list=['--service-name', '-n'],
             help="The name of the soft deleted API Management service instance.")
+
+    with self.argument_context('apim graphql api resolver create') as c:
+        c.argument('service_name', options_list=['--service-name', '-n'],
+                   help='The name of the API Management service instance.')
+        c.argument('api_id', arg_type=api_id, required=True)
+        c.argument('resolver_id', required=True,
+                   help='Resolver identifier within a GraphQL API. Must be unique in the current API Management service instance.')
+        c.argument('display_name',
+                   help='Resolver Name.', required=True)
```

Comment:
Please note that we can remove the default values of parameters in the custom methods to simplify the code configuration
```suggestion
        c.argument('service_name', options_list=['--service-name', '-n'],
                   help='The name of the API Management service instance.')
        c.argument('api_id', arg_type=api_id)
        c.argument('resolver_id',
                   help='Resolver identifier within a GraphQL API. Must be unique in the current API Management service instance.')
        c.argument('display_name',
                   help='Resolver Name.')
```